src/cpp/rtps/network/NetworkFactory.cpp

```diff
@@ -24,6 +22,7 @@
 #include <fastrtps/utils/IPLocator.h>
 
 #include <rtps/transport/UDPv4Transport.h>
+#include <rtps/network/NetworkFactory.h>
```

Comment:
In this case that it is the corresponding header file, it should be the first one included (even though it is private). See previous comment.